Formula/rustfmt.rb

```diff
@@ -20,17 +20,50 @@ class Rustfmt < Formula
 
   depends_on "rustup-init" => :build
   depends_on "rust" => :test
+  uses_from_macos "zlib"
+
+  on_linux do
+    depends_on "patchelf" => :build
+  end
```

Comment:
`patchelf.rb` should be available via `brew`, so we should try to use that instead of depending on `patchelf`. CC @Homebrew/linux for suggestions here.

Reply:
Do you mean the `patchelf.rb` formula or the API?